libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/ApolloClient.kt

```diff
@@ -550,6 +568,9 @@ private constructor(
                 if (webSocketReopenWhen != null) {
                   reopenWhen(webSocketReopenWhen)
                 }
+                if (webSocketReopenServerUrl != null) {
+                  serverUrl(webSocketReopenServerUrl)
```

Comment:
Should we throw here if `webSocketServerUrl != null && webSocketReopenServerUrl != null`?

Reply:
This is handled in `WebSocketNetworkTransport.Builder` but this reminds me we should add `webSocketReopenServerUrl` to the `copy()` function below

Reply:
Yup I was actually on it :)